source/agora/test/Base.d

```diff
@@ -1637,7 +1700,7 @@ public const(Block)[] getAllBlocks (TestAPI node)
     return blocks;
 }
 
-/// Returns: true if all the nodes contain the same blocks
+/// Returns: true if all the nodes contain the same block height
```

Comment:
Why? it does check the block contents too, not just the height.

Reply:
My mistake. Will revert this.